runtime/src/main/java/org/corfudb/runtime/view/LayoutSequencerHealingPolicy.java

```diff
@@ -0,0 +1,44 @@
+package org.corfudb.runtime.view;
+
+import java.util.Set;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.exceptions.LayoutModificationException;
+
+/**
+ * Handles the healing of responsive nodes.
+ *
+ * <p>Created by zlokhandwala on 11/21/16.
+ */
+public class LayoutSequencerHealingPolicy implements IReconfigurationHandlerPolicy {
```

Comment:
rename it to SequencerHealingPolicy as it only heals Sequencers.

Reply:
Done.